src/components/BurnWarning/style.ts

```diff
@@ -0,0 +1,69 @@
+import { makeStyles } from '@material-ui/core/styles'
+import { colors, typography } from '@static/theme'
+
+const useStyles = makeStyles(() => ({
+  warningContainer: {
```

Comment:
we already have utils for blurring, check other modals

Reply:
done

Reply:
now you can get rid of this